pages/Options.js

```diff
@@ -1,43 +1,67 @@
-import styled from "styled-components";
-import Link from "next/link";
 import {
+  Box,
   ContainerMain,
   ContainerMainLabel,
-  Box,
   InnereBox,
   SpaceLabel,
-} from "@/components/styles/container.styled";
+} from "@/components/styledComponents/Container.styled";
+
+import Link from "next/link";
+import { useState } from "react";
+
+const spk = "simple Sentences and for kids";
```

Comment:
It's strange that you have put this one option value into a variable while is directly in the JSX. Please consider having all options in an array and then mapping over that array in render.

Checkboxes are not the best way to represent a selection of one item. Please use a `<select>` element or radio buttons.



```js
const optionItems = [
  { label: "Simple Sentences & For Kids", value: "simple sentences and for kids"  },
  …
];
```

in the JSX:

```js
  <select name="options" onChange={(event) => handleOption(event.target.value)}>
    {optionItems.map((item) => (
      <option
        selected={options === item.value}
        value={item.value}
        key={item.value}
      >
        {item.label}
      </option>
    ))}
  </select>
```